modules/gatk/realignertargetcreator/main.nf

```diff
@@ -0,0 +1,43 @@
+process GATK_REALIGNERTARGETCREATOR {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk=3.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk:3.5--hdfd78af_11':
+        'quay.io/biocontainers/gatk:3.5--hdfd78af_11' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
```

Comment:
any intervals/target bed or other input formats then bam possible here?

Reply:
this is creating the intervals ;)